documentation/architecture/decisions/0005-deploying-prometheus-to-private-networks-in-aws.md

```diff
@@ -0,0 +1,59 @@
+# 1. Model for deploying prometheus to private networks in AWS
+
+Date: 2018-07-26
+
+## Status
+
+Draft
+
+## Context
+
+We are looking to deploy prometheus into private networks alongside
+existing infrastructure.  The existing infrastructure will be run by
+another team (called "client team" in this document), but our
+prometheus will be responsible for gathering metrics from it.
```

Comment:
Would be good to explain why we want to add federation so we can scrape their metrics rather than not.

I assume so it's that we will run nginx/alertmanager rather than requiring client teams to also have to do these things?

Reply:
I'm not sure I understand your question. Where does federation enter into it?

Reply:
I misunderstood 'our prometheus' to refer to our current prometheus that we are running at the moment rather than the one that would be deployed into the private network.

Would be good for context to expand on what you mean by 'prometheus'. Is that just prometheus or does it refer more to a prometheus monitoring service which would include potential things like nginx and alertmanager?